src/main/java/org/myrobotlab/service/HtmlFilter.java

```diff
@@ -111,7 +110,7 @@ public void setStripHtml(boolean stripHtml) {
 
   // helper function to strip html tags. 
   public static String stripHtml(String text) {
-    if (StringUtils.isEmpty(text))
+    if (text.isEmpty())

```

Comment:
This will cause a null pointer exception if text is null.  Please use StringUtils.isEmpty.